docs/howtos/socialnetwork/index-socialnetwork.mdx

```diff
@@ -8,7 +8,7 @@ authors: [julian,manuel]
 
 ![image](images/socialnetwork.png)
 
-In this blog post we’ll build a social network application using RediSearch and NodeJS. This is the idea that we used for our app [Skillmarket](https://www.youtube.com/watch?v=18NPKZy28cQ)
+In this blog post we’ll build a social network application using RediSearch and NodeJS. This is the idea that we used for our app [Skillmarket].(https://www.youtube.com/watch?v=18NPKZy28cQ)
```

Comment:
```suggestion
In this blog post we’ll build a social network application using RediSearch and NodeJS. This is the idea that we used for our app [Skillmarket](https://www.youtube.com/watch?v=18NPKZy28cQ).
```

Moved the full stop outside of the markdown link as this wasn't rendering properly on my local machine.

![Screen Shot 2021-11-24 at 5 34 36 PM](https://user-images.githubusercontent.com/94062/143287655-98da8106-f816-4268-83db-9b59226ef228.png)